docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-configuration.md

```diff
@@ -144,7 +145,7 @@ EOF
 ```
```

Comment:
Need to show `allowVolumeExpansion: true` in sample storage classes here. After the `provisioner` field.

Reply:
@avishnu Can you kindly provide your views on this.

Reply:
As discussed, this is not required to be documented in the command. @avishnu @dsharma-dc